package exception_handling_program_june6;

import java.io.*;
public class Test46 {
	public static void main(String[] args) throws Exception 
	{
		methodA();
	}

	public static void methodA() throws IOException
	{
		System.out.println("Completed..");
	}
}
